app/assets/javascripts/components/articles/available_article.jsx

```diff
@@ -70,6 +81,16 @@ const AvailableArticle = React.createClass({
       );
     }
 
+    // Shows editing suggestions only when suggestions are available for that class
+
+    let feedbackButton;
+    if (this.isEnglishWikipedia &&
+        !I18n.t(`suggestions.suggestion_docs.${assignment.article_rating}`).includes("missing")) {
```

Comment:
Also: `this.isEnglishWikipedia` is the function itself, rather than the return value of the function. Should be `this.isEnglishWikipedia()`.